python/tests/io/test_pyarrow.py

```diff
@@ -195,62 +195,6 @@ def test_raise_on_opening_a_local_file_not_found() -> None:
         assert "[Errno 2] Failed to open local file" in str(exc_info.value)
 
 
-def test_raise_on_opening_a_local_file_no_permission() -> None:
```

Comment:
I removed these tests since they don't work inside of a plain docker container because, in a docker container, everything runs under root unless specified otherwise. I don't feel like creating a special Docker image for this. These tests just assert the `PermissionError` that comes from Python itself.